app/src/main/java/com/mailerdaemon/app/placement/PlacementActivity.kt

```diff
@@ -73,6 +77,10 @@ class PlacementActivity : AppCompatActivity(), AccessDatabase {
                         PlacementAdapter.list = result
                         data = result
                         PlacementAdapter.notifyDataSetChanged()
+                        val bundle = Bundle()
+                        bundle.putParcelableArrayList("data", result as ArrayList<PlacementModel>)
+                        fragment.setArguments(bundle)
+                        supportFragmentManager.beginTransaction().replace(R.id.container, fragment).commit()
```

Comment:
call that interface method here instead of pushing fragment. Push fragment in onCreateView().